kdumpctl

```diff
@@ -650,6 +662,9 @@ load_kdump()
 
 	KEXEC_ARGS=$(prepare_kexec_args "${KEXEC_ARGS}")
 	KDUMP_COMMANDLINE=$(prepare_cmdline "${KDUMP_COMMANDLINE}" "${KDUMP_COMMANDLINE_REMOVE}" "${KDUMP_COMMANDLINE_APPEND}")
+	if is_aws_aarch64; then
+		KDUMP_COMMANDLINE=$(echo ${KDUMP_COMMANDLINE} | sed 's/irqpoll//' )
```

Comment:
You should use `remove_cmdline_param` provided by kdump-lib.sh to remove irqpoll because it could handle some corner cases like
 - there are multiple occurrences of irqpoll 
 - irqpoll is a sub string of another kernel parameter

Reply:
Note [this shellcheck warning](https://github.com/coiby/kexec-tools/runs/7350885398?check_suite_focus=true#step:3:458) could also be avoided if you use `remove_cmdline_param`.